requirements_dev.txt

```diff
@@ -1,4 +1,3 @@
-python-telegram-bot == 20.0a0
```

Comment:
It's duplicate because it's already in requirements.txt ?

Reply:
Yes, thought I removed it

Reply:
Yep, already removed in the next commit